update/qubes-vm.sls

```diff
@@ -5,12 +5,12 @@ dsa-4371-update:
     - source: salt://update/dsa-4371-update
     - runas: root
     - stateful: True
-{% endif %}
-
-{% if grains['os'] == 'Fedora' %}
-# workaround for https://bugzilla.redhat.com/1669247
-dnf list updates --refresh >/dev/null:
-  cmd.run
+{% elif grains['os_family'] == 'RedHat' %}
+dnf-makecache:
+  cmd.script:
+    - source: salt://update/dnf-makecache
+    - runas: root
+    - stateful: True
```

Comment:
Just curious - why stateful (which requires this extra script)? Contrary to `dsa-4371-update`, it isn't critical to block the other call if makecache fails. Just more friendly status, or something else?